### PR TITLE
Add mavshell example

### PR DESCRIPTION
**Pull request by JonasVautherin**
Add example for the shell plugin.

Also, somehow the examples/ CMakeLists was not a root project, so I fixed that.

### CODE REVIEW

File: example/mavshell/mavshell.cpp [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+int main(int argc, char** argv)
+{
+    if (!are_arguments_valid(argc, argv)) {
+        const auto binary_name = argv[0];
+        print_usage(binary_name);
+        return 1;
+    }
+
+    Mavsdk dc;
```
Comment (julianoes) [inline example/mavshell/mavshell.cpp]: Haha, just keep copying `dc`. It's not `DroneCore` anymore.

```suggestion
    Mavsdk mavsdk;
```

File: example/mavshell/mavshell.cpp [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    while (true) {
+        std::string command;
+        getline(std::cin, command);
+
+        if (command == "exit") {
+            break;
+        }
+
+        Shell::ShellMessage msg{true, timeout_ms, command + "\n"};
```
Comment (julianoes) [inline example/mavshell/mavshell.cpp]: I would probably add the newline inside the plugin, no?

Comment (irsdkv) [reply example/mavshell/mavshell.cpp]: @JonasVautherin Recently I was create [PR](https://github.com/mavlink/MAVSDK/pull/944) with that, so no more need to append `\n` :)

Comment (JonasVautherin) [reply example/mavshell/mavshell.cpp]: Agreed, I'll change that once the change is merged :slightly_smiling_face:

**Pull request merged**